bedrock/base/static.py

```diff
@@ -0,0 +1,17 @@
+from whitenoise.django import DjangoWhiteNoise
+
+
+class BedrockWhiteNoise(DjangoWhiteNoise):
+    """WhiteNoise class for modifying default cache headers."""
+    ONE_MONTH = 30 * 24 * 60 * 60
+
+    def add_cache_headers(self, static_file, url):
```

Comment:
This works in 1.0.6, but this method signature was changed in https://github.com/evansd/whitenoise/commit/108a0949e35ff59af49b98a52f19747464a2d7b4 -- not a blocker for merging this, but it's something we'll have to address when we upgrade.